charm-slurmctld/actions.yaml

```diff
@@ -45,3 +45,22 @@ influxdb-info:
 
     This action returns the host, port, username, password, database, and
     retention policy regarding to InfluxDB.
+
+etcd-get-root-password:
+  description: >
+    Get the password for the etcd root account.
+
+etcd-create-munge-account:
+  description: >
+    Create a new etcd account to be able to query the munge key.
+
+  params:
+    user:
```

Comment:
This is non-consequential really, but thought I should bring it up anyway..... 

Does it really matter what the username is here? Possibly we can be opinionated about the username and choose something up front. This way the user wouldn't have to decide and configure this additional param. Thoughts?

Reply:
What if we want different usernames for different uses? Lets say one for SimulationSoftwareX and a different one for SimulationSystemY. This way we can have a more granular control for accesses.